### PR TITLE
[UI] [AAP-6017] Disable save buttons on forms while submitting

### PR DESCRIPTION
**Pull request by lgalis**
- Disable the save button while submitting.
- Update api endpoint for project on the edit project page
- 

Screenshot from 2022-10-11 17-43-48

### CODE REVIEW

**Review by ddonahue007 — DISMISSED** [review]
lgtm

**Review by ddonahue007 — APPROVED** [review]
lgtm

File: ui/src/app/NewInventory/NewInventory.tsx [hunk 1]
```diff
@@ -195,7 +196,14 @@ const NewInventory: React.FunctionComponent = () => {
               </StackItem>
               <StackItem>
                 <ActionGroup>
-                  <Button variant="primary" isDisabled={true} onClick={handleSubmit}>Add</Button>
+                  <Button
+                    variant="primary"
+                    isDisabled={true}
```
Comment (eclarizio) [inline ui/src/app/NewInventory/NewInventory.tsx:201]: The rest of the buttons have `isDisabled` set to `{isSubmitting}`, why is this one hardcoded to `true`?

Comment (lgalis) [reply ui/src/app/NewInventory/NewInventory.tsx:201]: The andpoints for creating an inventory is not available yet. This one here is  placeholder. The rest of the pages have an actual working Add button

Comment (eclarizio) [reply ui/src/app/NewInventory/NewInventory.tsx:201]: Ah gotcha, makes sense 👍

**Review by eclarizio — APPROVED** [review]

**Pull request merged**